moveit_core/macros/include/moveit/macros/deprecation.h

```diff
@@ -36,8 +36,9 @@
 #define MOVEIT_MACROS_DEPRECATION_
 
 /** \def MOVEIT_DEPRECATED
-    Macro that marks functions as deprecated */
+    Deprecated macro that marks functions as deprecated */
```

Comment:
We deprecated the deprecator, lol

```suggestion
    Deprecated macro that marks functions as deprecated 
    
    TODO(Nov 2019): Delete this deprecated feature during the Noetic release. 
    */
```